deploy/local/run/docker-compose.yml

```diff
@@ -0,0 +1,68 @@
+# set the script version to 2.4 this is, so we can use feature
+# such as container health check & depends_on which is useful
+# for controlling the order of setting up the containers.
+version: '2.4'
+services:
+    mysql:
+        image: bitnami/mysql:5.7
+        ports:
+            - '3307:3306'
```

Comment:
The default port for MySQL is `3306`; if the developer already installed MySQL on their computer, this will lead to port conflict. This is why I set the publish port for the service to be `3307`. By doing this the developer doesn't need to kill their MySQL service on their computer.